### ISSUE TITLE
Installing on Synology NAS however not performing tests

### ISSUE THREAD

**Issue by JDMils** [henrywhitaker3/Speedtest-Tracker#1072]
I'm using the instructions found here (https://david.gardiner.net.au/2022/01/synology-speedtest.html) to install the container into a Docker environment on my Synology NAS and the page loads however the tests don't run successfully. I keep getting the following for All Tests:
ID=26, Time=InvalidDate, Download=x, Upload=x, Ping=x

Not sure if this helps, but when viewing the Log from the Docker container, I see the following:
sh: /config/www/app/Bin/speedtest: not found

Is there a way to get more info on what's going wrong?

**Comment by ks2018ks** [comment]
I have the same problem.

**Comment by spcqike** [comment]
Maybe check out this link 
https://github.com/henrywhitaker3/Speedtest-Tracker/issues/1062#issuecomment-1265361824

**Comment by ks2018ks** [comment]
> Maybe check out this link [#1062 (comment)](https://github.com/henrywhitaker3/Speedtest-Tracker/issues/1062#issuecomment-1265361824)

Looks like that other one doesn't do graphs.  I don't really want to have to install/update more applications.  Would rather just install one

**Comment by rob-1981** [comment]
https://mariushosting.com/how-to-install-speedtest-tracker-on-your-synology-nas/

**Comment by ks2018ks** [comment]
Yes, that's how I installed it, but the scheduled speed tests don't work. All of them fail with a red x except the first few manual tests
